data/releases/schedule.yaml

```diff
@@ -129,4 +141,4 @@ schedules:
       targetDate: 2022-05-24
     - release: 1.24.0
       cherryPickDeadline: ""
-      targetDate: 2022-05-03
+      targetDate: 2022-05-03
```

Comment:
Nit: please return the new line.

Reply:
Good catch - fixed here and opened up #40835 because I was very confused why this was happening since my editor normally doesn't go rogue and strip random final LF chars 😅